optimum/intel/openvino/modeling_visual_language.py

```diff
@@ -1294,7 +1336,7 @@ def merge_vision_text_embeddings(
             final_attention_mask |= image_to_overwrite
             position_ids = (final_attention_mask.cumsum(-1) - 1).masked_fill_((final_attention_mask == 0), 1)
         else:
-            special_image_mask = (input_ids == self.config.image_token_index).unsqueeze(-1).expand_as(inputs_embeds)
+            special_image_mask = torch.tensor((input_ids == image_token_index)).unsqueeze(-1).expand_as(inputs_embeds)
```

Comment:
is it necessary to apply `torch.tensor` here ?

Reply:
no, added during debugging, will return back